README.md

```diff
@@ -4,7 +4,7 @@
 - This is a sample project for demonstration purposes.   
 - The project demonstrates how to use [Spring Cloud Azure](https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html) to develop applications.    
 - The sample project contains 42 samples, and it still grows.    
-- These samples are grouped by Azure services and [Spring Cloud Azure libraries](https://github.com/Azure/azure-sdk-for-java/tree/feature/azure-spring-cloud-4.0/sdk/spring).    
+- These samples are grouped by Azure services and [Spring Cloud Azure libraries](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring).    
```

Comment:
Do you confirm all links have updated to main branch?

Reply:
No, just fix the broken links here.
The main scope of this PR is to update github actions.